packages/calling-stateful-client/review/beta/calling-stateful-client.api.md

```diff
@@ -193,7 +196,7 @@ export interface RemoteVideoStreamState {
 
 // @public
 export interface StatefulCallClient extends CallClient {
-    createCallAgent(...args: Parameters<CallClient['createCallAgent']>): Promise<DeclarativeCallAgent>;
+    createCallAgent(tokenCredential: CommunicationTokenCredential, options?: CallAgentOptions, alternativeCallerId?: string): Promise<DeclarativeCallAgent>;
```

Comment:
Why are we completely changing the function signature here?

Reply:
@dmceachernmsft could you possibly reach out to someone from ARB and see if they consider this a change?

Reply:
I've asked a similar question before. As long as the final concrete types are unchanged, it is not considered a breaking change.